hummingbot/connector/exchange/bloxroute_openbook/bloxroute_openbook_exchange.py

```diff
@@ -72,7 +69,7 @@ def __init__(
         self._sol_wallet_private_key = solana_wallet_private_key
 
         self._key_pair = load_private_key(self._sol_wallet_private_key)
-        self._sol_wallet_public_key = str(self._key_pair.pubkey())
+        self._sol_wallet_public_key = str(self._key_pair.public_key)
```

Comment:
@kdonthi 

https://github.com/bloXroute-Labs/solana-trader-client-python/commit/70dfdd7f17cf4ebec7341d07e67afbd1ce212d89

I think if you use bxsolana-trader-1.5.0, you will see that `public_key` is deprecated and it is using pubkey() now?